cmd/agent/subcommands/status/command.go

```diff
@@ -138,18 +129,7 @@ func setIpcURL(cliParams *cliParams) url.Values {
 }
 
 func renderResponse(res []byte, cliParams *cliParams) error {
-	var s string
-
-	// The rendering is done in the client so that the agent has less work to do
-	if cliParams.prettyPrintJSON {
-		var prettyJSON bytes.Buffer
-		json.Indent(&prettyJSON, res, "", "  ") //nolint:errcheck
-		s = prettyJSON.String()
-	} else if cliParams.jsonStatus {
-		s = string(res)
-	} else {
-		s = scrubMessage(string(res))
-	}
```

Comment:
Can we wait for @GustavoCaso on that one ? 
IIRC the status used to be rendered in the server but we moved it to the client to fix some bug, so I'm afraid we would just revert that...

Reply:
Actually it could have been the `check` subcommand, I'm not sure 😅

Reply:
Yes for sure 🙇🏻